js/main.js

```diff
@@ -1133,7 +1138,7 @@ function renderGraph() {
 					tooltipTr = tooltipTable.append("tr");
 					if (d.produce.jarType) {
 						tooltipTr.append("td").attr("class", "tooltipTdLeftSpace").text("Value (" + d.produce.jarType + "):");
-						tooltipTr.append("td").attr("class", "tooltipTdRight").text(d.produce.price * 2 + 50)
+						tooltipTr.append("td").attr("class", "tooltipTdRight").text(options.skills.arti ? Math.round((d.produce.price * 2 + 50) * 1.4) : d.produce.price * 2 + 50)
```

Comment:
I'd say to keep this in the other pull request just to avoid confusion and not mix two distinct issues into one.
On that note, maybe it would make sense to change the name of this pull request to encompass the wider 'keg calculations' problem